melos.yaml

```diff
@@ -51,6 +51,10 @@ scripts:
     run: melos exec flutter pub run dartdoc
     description: Run dartdoc checks for all packages.
 
+  sphinxdoc:
+    run: cd doc/_sphinx && make html
```

Comment:
Is there a way to make this work even if you're currently in a subdirectory? Say, if `cwd = packages/flame`?

Reply:
Yes, but first of all the makefile should be rewritten to depend on paths in relation to where the file is, not where the shell is.

Reply:
Fixed it so that it can run from any directory